src/pretix/settings.py

```diff
@@ -116,6 +115,29 @@
 
 db_options = {}
 
+postgresql_sslmode = config.get('database', 'sslmode')
+USE_DATABASE_TLS = True if postgresql_sslmode is not None and postgresql_sslmode != "disable" else False
```

Comment:
This could be simplified with

```suggestion
postgresql_sslmode = config.get('database', 'sslmode', fallback="disable")
USE_DATABASE_TLS = postgresql_sslmode != "disable"
```